addon/utils.js

```diff
@@ -390,6 +382,22 @@ export function getErrorMessage (error) {
 }
 /* eslint-enable complexity */
 
+export function getSubModel (pathSegment, bunsenModel) {
+  if (/^\d+$/.test(pathSegment)) {
+    bunsenModel = bunsenModel.items
+
+    if (Array.isArray(bunsenModel)) {
+      bunsenModel = bunsenModel[parseInt(pathSegment)]
+    }
+  } else if ('properties' in bunsenModel) {
+    bunsenModel = bunsenModel.properties[pathSegment]
+  } else {
+    return undefined
```

Comment:
This is a condition doesn't seem to exist in the original code. Do we have a use case in bunsen that we would want this condition to be hit?

Reply:
I hit it. That's why I added it. If the path doesn't match the model, then this can happen.

```
type: 'object'
properties: {
  a: {type: 'string'}
}
```

Works when the path is `a`. Doesn't work when the model is `a.b`.